CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Changed
 - add regex and option to negate conditional profiles search
 
+### Fixed
+- empty profile name when polling varbind that was not node
```

Comment:
maybe - when MIB family name and a polled varbind differs?

Reply:
sounds better 😄